content/news/2024-07-15/392/index.fr.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Avez-vous déjà installé la version de juillet d'Organic Maps ? Elle introduit deux fonctionnalités majeures financées par le fonds d'investissement de l'NGI0"
+date: 2024-07-15T21:13:23+00:00
+slug: july-release-with-improved-usa-address-search-using-tiger-data-and-fixed-text-rendering-for-non-latin-scripts-funded-by-ngi0-entrust-fund
+aliases: ["/fr/news/2024-07-15/avez-vous-déjà-installé-une-version-de-juillet-d-organic-maps-qui-instaure-deux-fonctionnalités-majoritaires-financées-par-le-fonds-dinvestissement-de-l-ngi0/"]
```

Comment:
Why the alias is needed? Is there already an existing news link with that address?

Reply:
It's not necessary, it's deleted.